hazelcast/connection.py

```diff
@@ -45,6 +46,11 @@ def __init__(self, initial_backoff, max_backoff, multiplier, cluster_connect_tim
         self._cluster_connect_attempt_begin = None
         self._current_backoff = None
 
+        if cluster_connect_timeout == _INF:
+            self._cluster_connect_timeout_text = "INFINITE"
+        else:
+            self._cluster_connect_timeout_text = "%ds" % self._cluster_connect_timeout
```

Comment:
Should we format the timeout using something like `%.2f` in case someone uses a fractional timeout?